ec2api/api/subnet.py

```diff
@@ -111,8 +111,19 @@ def create_subnet(context, vpc_id, cidr_block,
 
     gateway_ip = str(netaddr.IPAddress(subnet_ipnet.first + 1))
     main_route_table = db_api.get_item_by_id(context, vpc['route_table_id'])
-    host_routes = route_table_api._get_subnet_host_routes(
-            context, main_route_table, gateway_ip)
+
+    #Get vpc and subnet cidr range
+    subnet_cidr_range = int(cidr_block.split('/')[1])
+    vpc_cidr_range = int(vpc['cidr_block'].split('/')[1])
```

Comment:
Sure. I had spotted that but I just followed the method that was used in format_subnet. Ill change the same.